homemate/src/components/newPost/index.js

```diff
@@ -0,0 +1,197 @@
+import React, { useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+
+import api from '../../api';
+import Input from '../input';
+import BaseButton from '../button';
+import Loading from '../loading';
+
+import './newPost.css';
```

Comment:
```suggestion
import React, { useState } from 'react';
import { toast } from 'react-toastify';
import { Tooltip } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';

import api from '../../api';
import Input from '../input';
import BaseButton from '../button';
import Loading from '../loading';
import useFetch from '../../hooks/useFetch';
```

retirando imports desnecessarios e importanto coisas q irei usar abaixo